test/cpp/cpp_test_util.cpp

```diff
@@ -307,7 +307,7 @@ std::vector<at::Tensor> Fetch(
   std::vector<at::Tensor> tensors;
   for (auto& literal : literals) {
     tensors.push_back(MakeTensorFromXlaLiteral(
-        literal, TensorTypeFromXlaType(literal.shape().element_type())));
+        literal, MaybeUpcastForHost(literal.shape().element_type())));
```

Comment:
I understand the motivation of this renaming but it made it less obvious to see this function does a conversation from XLA type to Tensor type. `MaybeUpcastXlaTypeToTensorType`? Same for the downcast.

Reply:
Done. Renamed to `MaybeUpcastToHostTorchType` and `MaybeDowncastToXlaDeviceType` to clarify the return type.